docs/src/StraightLinePrograms/intro.md

```diff
@@ -205,3 +205,10 @@ false
 ```
 
 
+## Contact
+
+Please direct questions about this part of OSCAR to the following people:
+* [Raphael Fourquet](https://github.com/rfourquet),
```

Comment:
Not sure we should point to @rfourquet herr - at the very least we should ask him if he'd be comfortable with that. After all, he is not being to work on this anymore, and while he *might* be willing to provide support, this is not automatic.

Reply:
Agreed.

For similar reasons, let me ping @LukasKuehne to find out if he is comfortable with being listed as contact for the combinatorics section.

Reply:
@LukasKuehne and @bschroter Should there be a section about matroids?